pox/lib/addresses.py

```diff
@@ -436,7 +463,7 @@ def from_num (cls, num):
     """
     o = b''
     for i in range(16):
-      o = chr(num & 0xff) + o
+      o = chr(num & 0xff).encode('latin-1') + o
```

Comment:
I think we should probably just avoid chr().  I don't think there's a direct equivalent for bytes objects, but we could make a chrb or something and put it in util or something.  Maybe along the lines of `chrb = lambda x: struct.pack('B', x)` ?

Reply:
This should work, but the notion of `chr` changes a bit in python3 (i.e. it's about unicode specifically) so I would call it something different.